.github/workflows/build.yml

```
@@ -159,7 +159,29 @@ jobs:
 
       - name: Configure project
         run: |
-          cmake --preset "$CMAKE_PRESET"
+          cmake --preset "$CMAKE_PRESET" -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
```

```suggestion
          cmake --preset "$CMAKE_PRESET"
```
https://github.com/PrismLauncher/PrismLauncher/blob/ff98c350a1c09f84faf7a2e08af7c50175f23b62/CMakeLists.txt#L110-L114